packages/contracts-bedrock/scripts/deploy/VerifyOPCM.s.sol

```diff
@@ -116,6 +122,22 @@ contract VerifyOPCM is Script {
         sourceNameOverrides["OPContractsManagerUpgrader"] = "OPContractsManager";
         sourceNameOverrides["OPContractsManagerInteropMigrator"] = "OPContractsManager";
 
+        // Expected getter functions that should be checked in OPCM verification.
+        // Any getter in the ABI that's not in this list will cause verification to fail.
```

Comment:
Change this comment here to avoid people just setting this to true

Reply:
(From Kelvin)

Reply:
> Change this comment here to avoid people just setting this to true

Agree. My concern is that people will just add stuff here in a rush, and not realize they need to add it to `_verifyOpcmImmutableVariables()` below. 

One idea could be to have this mapping point to a string. If the string is `"SKIP"`, then it's assumed that the value is verified elsewhere, otherwise the string should be an env var name that is used to look up the expected value. 

That would allow us to have just this one mapping to maintain.

Reply:
Solved in commit 28e62d0 by adding the solution you mentioned with a mapping, I've also added more comments to make sure people understand it when they are changing it